llm/deepseek-r1/README.md

```diff
@@ -62,14 +61,30 @@ run: |
 
 ```
 
+Adjust the `accelerators` and `num_nodes` to fit your needs. Common configurations include:
```

Comment:
Nice! How about we keep this table after the command line below and instead of asking people to change the yaml, use the inline cli flags.

Reply:
Great suggestion! Moved the table after the CLI command and updated the instructions to use inline flags instead of modifying YAML. Thanks for the improvement!